brickstrap.sh

```diff
@@ -214,9 +238,10 @@ function brp_sanity_check_cli_options()
 
 function brp_validate_cli_options()
 {
-    brp_sanity_check_cli_options "$@" && \
-    brp_validate_project_name && \
+    brp_sanity_check_cli_options "$@"
+    brp_validate_project_name
     brp_validate_component_names
+    brp_validate_qemu || [ $? -eq 255 ] # no QEMU specified = 255
```

Comment:
It seems that `brp_validate_qemu` will allow anything other than an empty string when specifying the `-Q` option. Is that the intended behavior? It seems like it should check that an actual matching binary could be found at this point. Otherwise, you get all the way through downloading and unpacking packages before brickstrap fails, which wastes lots of time.